local-storage/src/LocalStorage.ts

```diff
@@ -0,0 +1,97 @@
+import { CurriedFunction2, curry2 } from '@most/prelude';
+import {
+  LocalStorageClearCommand,
+  LocalStorageCommand,
+  LocalStorageRemoveItemCommand,
+  LocalStorageSetItemCommand,
+  LocalStorageSource,
+} from './types';
+import {
+  Stream,
+  constant,
+  drain,
+  filter,
+  fromEvent,
+  map,
+  merge,
+  startWith,
+} from 'most';
+
+import { hold } from '@most/hold';
+
+export interface LocalStorageSinks {
+  localStorage$: Stream<LocalStorageCommand>;
+}
+
+export interface LocalStorageSources {
+  localStorage: LocalStorageSource;
+}
+
+export function LocalStorage(sinks: LocalStorageSinks): LocalStorageSources {
```

Comment:
Hello @milankinen, the README example is in fact correct if you take closer look at it. 
Instead of thinking in terms of "drivers", since `@motorcycle/run` no longer has a magical "drivers object", we're now transitioning to "everything is a component`, which really means "everything is a function". 

Typically components in Cycle always go `Sources -> Sinks`, and now I'm loosing coining the opposite, like we have here, `Effectful Components` as `Sinks -> Sources`.